deploy/007-OVM_L1CrossDomainMessenger.deploy.ts

```diff
@@ -0,0 +1,59 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+/* Imports: Internal */
+import { getDeployedContract } from '../src/hardhat-deploy-ethers'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deploy } = hre.deployments
+  const { deployer } = await hre.getNamedAccounts()
+
+  const Lib_AddressManager = await getDeployedContract(
+    hre,
+    'Lib_AddressManager',
+    {
+      signerOrProvider: deployer,
+    }
+  )
+
+  const result = await deploy('OVM_L1CrossDomainMessenger', {
+    from: deployer,
+    args: [],
+    log: true,
+  })
+
+  if (!result.newlyDeployed) {
+    return
+  }
+
+  const OVM_L1CrossDomainMessenger = await getDeployedContract(
+    hre,
+    'OVM_L1CrossDomainMessenger',
+    {
+      signerOrProvider: deployer,
+    }
+  )
+
+  await OVM_L1CrossDomainMessenger.initialize(Lib_AddressManager.address)
```

Comment:
can we add a clarifying comment here that this is technically not required?  Just for auditability